lib/components/FileAttachment/index.jsx

```diff
@@ -0,0 +1,181 @@
+  const [error, setError] = useState("");
+  const uppyConfig = { ...DEFAULT_UPPY_CONFIG };
+  const onClickAttachment = () => fileInputRef?.current?.click();
+
+  const {
+    uppy,
+    isUploading,
+    error: uppyError,
+  } = useUppyUploader({
+    endpoint,
+    uppyConfig,
+    uppyUploadConfig: { ...UPPY_UPLOAD_CONFIG },
+  });
+
+  useEffect(() => {
+    if (!isEmpty(uppyError)) {
+      setError(uppyError);
+    }
+  }, [uppyError]);
+
+  const onSelectAttachment = async e => {
+    const files = Array.from(e?.target.files);
+    try {
+      for (const file of files) {
+        uppy.addFile({
+          name: file.name,
+          type: file.type,
+          data: file,
+        });
+      }
+
+      const { successful = [], failed = [] } = await uppy.upload();
+
+      if (!isEmpty(successful)) {
+        setUploadedFiles([...uploadedFiles, ...successful]);
+      }
+
+      if (failed.length > 0) {
+        const errorMessages = failed.map(file => file?.error.message).join(",");
+        setError(errorMessages);
+      }
+    } catch (e) {
+      onError && onError(e);
+    }
+  };
+
+  const handleDelete = async file => {
+    const { signed_id: signedId } = file.response;
+
+    try {
+      await directUploadsApi.destroy(`${endpoint}/${signedId}`);
+      setUploadedFiles(
+        uploadedFiles.filter(
+          uploadedFile => uploadedFile.response.signed_id !== signedId
+        )
+      );
+      onDelete(file);
+    } catch (e) {
+      setError(`Something went wrong while deleting file: ${file.name}`);
+      onError && onError(e);
+    }
+  };
+
+  const handleRename = async (attachment, callback) => {
+    try {
+      const { fileName, signedId } = attachment;
+      const payload = {
+        blob: { filename: fileName },
+      };
+
+      const response = await directUploadsApi.create(
+        `${endpoint}/${signedId}/`,
+        payload
+      );
+      const { blob = {} } = response.data;
+
+      if (!isEmpty(blob)) {
+        const updatedFiles = uploadedFiles.map(file =>
+          file.response.signed_id === signedId
+            ? { ...file, name: blob?.filename }
+            : file
+        );
+        setUploadedFiles([...updatedFiles]);
+        callback(true);
+        onRename(blob.filename);
+      }
+    } catch (e) {
+      onError && onError(e);
+    }
+  };
+
+  const handleView = ({ response, name }) => {
+    if (response && response.blob_url) {
+      saveAs(response.blob_url, name);
+    }
+  };
+
+  const dropDownOptions = [
+    {
+      key: DROP_DOWN_OPTIONS_KEYS.VIEW,
+      label: DROP_DOWN_OPTIONS_KEYS.VIEW,
+      handler: handleView,
+    },
+    {
+      key: DROP_DOWN_OPTIONS_KEYS.DELETE,
+      label: DROP_DOWN_OPTIONS_KEYS.DELETE,
+      handler: handleDelete,
+    },
+    {
+      key: DROP_DOWN_OPTIONS_KEYS.RENAME,
+      label: DROP_DOWN_OPTIONS_KEYS.RENAME,
+      handler: null,
+    },
+  ];
+
+  return (
+    <div className="neeto-editor-attachment-uploader__dnd-wrapper">
```

Comment:
What is the need for this class? I don't see the definition anywhere. Please remove all such unused classes.

If you are defining a new class please update `neeto-editor-` prefix to `ne-`. We are slowly moving to this style.